tests/v3/test_store/test_stateful_store.py

```diff
@@ -216,6 +226,8 @@ def check_num_keys_equal(self) -> None:
     def check_keys(self) -> None:
         keys = list(self.store.list())
 
+        # NOTE: A local store can be non-empty if there are no files,
+        # but there are subfolders that weren't cleared.
```

Comment:
hah this is wild. What does it mean for a `LocalStore` to be empty?

Reply:
this seems like a bug. The expected behavior is to ignore all assets (including empty directories) that are not recognized parts of the hierarchy.

Reply:
Opened #2218 .